### ISSUE TITLE
Vite 4 support

### ISSUE THREAD

**Issue by alensiljak** [vite-pwa/sveltekit#17]
SvelteKit is finally 1.0.0. Alas, adding the PWA functionality fails again. :'(

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: cashier-sveltekit@0.0.1
npm ERR! Found: vite@4.0.1
npm ERR! node_modules/vite
npm ERR!   dev vite@"^4.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"^3.1.0" from vite-plugin-pwa@0.13.3
npm ERR! node_modules/vite-plugin-pwa
npm ERR!   peer vite-plugin-pwa@"^0.13.3" from @vite-pwa/sveltekit@0.0.1
npm ERR!   node_modules/@vite-pwa/sveltekit
npm ERR!     dev @vite-pwa/sveltekit@"0.0.1" from the root project
```

Is Vite 4 support in the pipelines?
Thanks, guys!

**Comment by userquin** [comment]
@alensiljak yes, we're working on it, we need to apply a few changes.

**Comment by userquin** [comment]
released 0.1.0 version

**Comment by alensiljak** [comment]
Thank you for the present! Looking forward to using it this weekend.
